app/features/notifications/services/fakeFirebase.js

```diff
@@ -0,0 +1,29 @@
+import { noop } from 'lodash'
+
+function fakeSubscribe() {
+  return noop;
+}
+
+function notifications() {
```

Comment:
I'd prefer simple checks through code  if (canWeSupportNotifications) { ... 
Instead of fake something. 
So, not need in this module.